### ISSUE TITLE
Adjust HTML in a11y skip link

### ISSUE THREAD

**Issue by knice** [ucsc/ucsc-2022#293]
https://ucsc.service-now.com/now/nav/ui/classic/params/target/incident.do%3Fsys_id%3D2df010581b93f510931886e0604bcbca%26sysparm_stack%3D%26sysparm_view%3D

**Comment by knice** [comment]
The skip link is in `functions.php`. It was added by Modern Tribe. I think it is getting flagged because it is just an `a` tag which is an inline element. The link needs a block-level wrapper element with a proper ARIA role to make it a landmark role (landmark roles are the only roles that can be direct descendants of the `body` tag). 

You can refer to the skip link functionality we have on all WCMS pages for reference.

**Comment by knice** [comment]
Additionally, the skip link nav was being added just above the page content wrapper. A11y testing tools flagged that as an issue. The skip link should be as close to the top of the page as possible.
